sdk/blockchain-client-provider/src/implementation/BlockchainClientProvider.ts

```diff
@@ -0,0 +1,67 @@
+import { createPublicClient, http, type Chain } from 'viem'
+import { arbitrum, base, mainnet, optimism } from 'viem/chains'
+
+import { IConfigurationProvider } from '@summerfi/configuration-provider'
+import type { IChainInfo } from '@summerfi/sdk-common'
+import {
+  IRpcConfig,
+  getRpcGatewayEndpoint,
+} from '@summerfi/serverless-shared/getRpcGatewayEndpoint'
+import type { BlockchainClient } from '../types/BlockchainClient'
+import { IBlockchainClientProvider } from '../interfaces'
+
+/**
+ * RPC configuration for the RPC Gateway
+ */
+const rpcConfig: IRpcConfig = {
+  skipCache: false,
+  skipMulticall: false,
+  skipGraph: true,
+  stage: 'prod',
+  source: 'borrow-prod',
+}
+
```

Comment:
Add `// @see IBlockchainClientProvider` as comment to inherit the docs from the interface

Reply:
I think // @implements IBlockchainClientProvider would work the same but is more appropriate, WDYT?